docs_src/security/secrets-config-proxy.md

```diff
@@ -18,10 +18,13 @@ Proxy configuration commands (listed below) require access to the secret store m
 
 # OPTIONS
 
-  * **--confdir** _/path/to/directory/with/configuration.toml_ (optional)
+  * **--configDir** _/path/to/directory/with/configuration.toml_ (optional)
```

Comment:
We're going to be dropping TOML support in v3.0, so maybe it's time to make this comment more general?

Reply:
@TomBrennan-Eaton , a larger Docs PR will be needed once we make to switch away from TOML. Best to make all those changes together and wait until we have made the actual code changes.